terms/neuro/study.json

```
@@ -522,6 +522,10 @@
             "const": "scRNA-seq_wild_AD_mic",
             "description": "Microglia of wild-derived Alzheimer's mouse study (scRNA-seq_wild_AD_mic)",
             "source": "https://www.synapse.org/#!Synapse:syn23763501"
-        }
-    ]
+        },
+         {
+            "const": "HBI_scRNAseq",
+            "description": "The human brain immunity scRNAseq study (HBI_scRNAseq)",
+            "source": "https://www.synapse.org/#!Synapse:syn24168322"
+        }    ]
```

```suggestion
        }
    ]
```

Just for ease of reading I'd suggest keeping the formatting of the square bracket on its own line

---

Noted! Thanks.